File: 백준/Silver/11399. ATM/ATM.cs
using System;
using System.Linq;

class Program
{
    static void Main(string [] args)
    {
        int N = int.Parse(Console.ReadLine());
        int [] array = Console.ReadLine().Split(' ').Select(int.Parse).ToArray();
        
        Array.Sort(array);
        
        int sum = 0;
        int answer = 0;
        
        for(int i=0; i<N; i++)
        {
            sum += array[i];
            answer += sum;
        }
        Console.WriteLine(answer);
    }
}

File: 백준/Silver/11399. ATM/ATM.js
let input = require('fs')
.readFileSync('/dev/stdin')
.toString().split('\n');

let N = Number(input[0]);
let arr = input[1].split(' ').map(Number);

arr.sort(function(a,b){
    return a - b;
})
let sum = 0;
let answer = 0;
for (let i = 0; i<N; i++)
{
    sum += arr[i];
    answer += sum;
}

console.log(answer);